BraveRewardsUI/Publisher/PublisherView.swift

```diff
@@ -84,7 +86,12 @@ class PublisherView: UIStackView {
     $0.adjustsFontSizeToFitWidth = true
   }
   // Only shown when unverified
-  private let unverifiedDisclaimerView = DisclaimerView(text: Strings.UnverifiedPublisherDisclaimer)
+  private let unverifiedDisclaimerView = LinkLabel().then {
+    $0.textColor = Colors.grey200
+    $0.font = UIFont.systemFont(ofSize: 12.0)
+    $0.textContainerInset = UIEdgeInsets(top: 8.0, left: 8.0, bottom: 8.0, right: 8.0)
+    $0.text = "\(Strings.UnverifiedPublisherDisclaimer) \(Strings.DisclaimerLearnMore)"
```

Comment:
Missing background color + radius here, sorry missed this one, its the same as the other replaced DisclaimerView (`UIColor(white: 0.0, alpha: 0.04)`)

![Simulator Screen Shot - iPhone 8 - 2019-06-25 at 16 40 08](https://user-images.githubusercontent.com/529104/60131853-4ca84580-9768-11e9-9a5b-c6e6769cfa55.png)